modules/candidate_parameters/jsx/monitoring.js

```diff
@@ -215,41 +215,77 @@ var MonitoringStatus = React.createClass(
       var formattedHistory = [];
       for (let row in this.state.Data.history) {
         if (this.state.Data.history.hasOwnProperty(row)) {
-          var line = "";
-          var lineInfo = "";
-          var historyRow = this.state.Data.history[row];
-          var userH = historyRow.entry_staff;
-          var updateDateH = historyRow.data_entry_date;
-          var visitH = historyRow.visit_label;
-          var flaggedH = historyRow.flag;
-          var monitoredH = historyRow.monitored;
-          var monitoredDateH = historyRow.date_monitored;
-          var examinerH = historyRow.monitor_id;
+          let line = [];
+          let lineInfo = "";
+          let historyRow = this.state.Data.history[row];
+          let userH = historyRow.entry_staff;
+          let updateDateH = historyRow.data_entry_date;
+          let visitH = historyRow.visit_label;
+          let flaggedH = historyRow.flag;
+          let monitoredH = historyRow.monitored;
+          let monitoredDateH = historyRow.date_monitored;
+          let examinerH = historyRow.monitor_id;
 
           lineInfo += "[UPDATE by ";
           lineInfo += userH;
           lineInfo += " at ";
           lineInfo += updateDateH;
-          lineInfo += "] ";
-          line += " Visit -> ";
-          line += visitH;
-          line += " Flagged -> ";
-          line += flaggedH;
-          line += " Monitored -> ";
-          line += monitoredH;
+          lineInfo += "]";
 
+          line.push(
+            <td>
+              <b>
+                Visit->
+              </b>
+              {visitH}
+            </td>
+          );
+          line.push(
+            <td class="td-hist">
+              <b>
+                Flagged ->
+              </b>
+              {flaggedH}
+            </td>
+          );
+          line.push(
+            <td class="td-hist">
+              <b>
+                Monitored ->
+              </b>
+              {monitoredH}
+            </td>
+          );
           if (monitoredDateH !== null) {
-            line += " Date Monitored -> ";
-            line += monitoredDateH;
-            line += "   ";
+            line.push(
+              <td class="td-hist">
+                <b>
```

Comment:
Not critical, but when content is short, it is generally easier to read in 1 line (aka `<b>Date Monitored -></b>`) makes code more condensed/easier to read)